CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.10.3 (2020-02-20)
+
+### Features
+- New `date-input` component [raul-onna]
+
```

Comment:
you also need add a section bugfixes to list the bug fixes you did in other components (I remember you did a fix in calendar component, didn't you?)

Reply:
Totally right, I forgot it :)